lib/ansible/plugins/lookup/password.py

```diff
@@ -276,10 +279,32 @@ def run(self, terms, variables, **kwargs):
             b_path = to_bytes(path, errors='surrogate_or_strict')
             chars = _gen_candidate_chars(params['chars'])
 
+            content = None
             changed = False
-            content = _read_password_file(b_path)
-
-            if content is None or b_path == to_bytes('/dev/null'):
+            writer_process = False
+            if not os.path.exists(b_path) and b_path != to_bytes('/dev/null'):
```

Comment:
So, if b_path is '/dev/null', why do we have the 2nd test, and in which case would /dev/null be inexistant ?